sa/_db-next/migrations/20171214114506_AddOrderFQDNSets.sql

```diff
@@ -0,0 +1,27 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE orderFqdnSets (
+  id BIGINT(20) NOT NULL AUTO_INCREMENT,
+  setHash BINARY(32) NOT NULL,
+  orderID BIGINT(20) NOT NULL,
+  registrationID BIGINT(20) NOT NULL,
+  expires DATETIME NOT NULL,
+  PRIMARY KEY (id),
+  KEY setHash_expires_idx (setHash,expires),
+  CONSTRAINT orderFqdnSets_registrationID_registrations
+    FOREIGN KEY (registrationID)
+    REFERENCES registrations (id)
+    ON DELETE NO ACTION ON UPDATE NO ACTION,
+  CONSTRAINT orderFqdnSets_orderID_orders
+    FOREIGN KEY (orderID)
+    REFERENCES orders (id)
+    ON DELETE NO ACTION ON UPDATE NO ACTION
```

Comment:
Ditto here, if the order doesn't exist we don't want to be able to look it up here.